src/components/v5/common/Navigation/ColonySidebar/partials/RouteSection/partials/RouteSectionItem/RouteSectionItem.tsx

```diff
@@ -0,0 +1,109 @@
+import { CaretDown } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import { AnimatePresence, motion } from 'framer-motion';
+import React, { useState } from 'react';
+import { useMatch, useNavigate } from 'react-router-dom';
+
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { formatText } from '~utils/intl.ts';
+import {
+  sidebarButtonIconStyles,
+  sidebarButtonStyles,
+  sidebarButtonTextStyles,
+} from '~v5/common/Navigation/consts.ts';
+import { usePageLayoutContext } from '~v5/frame/PageLayout/context/PageLayoutContext.ts';
+import Button from '~v5/shared/Button/Button.tsx';
+
+import { motionVariants } from './consts.ts';
+import { type RouteSectionItemProps } from './types.ts';
+
+const displayName =
+  'v4.frame.NavigationSidebar.partials.NavigationSection.partials.NavigationItem';
+
+const NavigationSectionItem: React.FC<RouteSectionItemProps> = ({
+  subItems,
+  translation,
+  path,
+  icon: Icon,
+  routeType,
+}) => {
+  const { colony } = useColonyContext();
+
+  const navigate = useNavigate();
+
+  const matchingRoute = useMatch(`${colony.name}/${path}`);
+
+  const [isAccordionExpanded, setIsAccordionExpanded] = useState(false);
+
+  const { toggleSidebar } = usePageLayoutContext();
+
+  const isAccordion = !!subItems;
+
+  const handleOnClick = () => {
+    if (isAccordion) {
+      setIsAccordionExpanded((state) => !state);
+    }
+
+    if (path && routeType) {
+      toggleSidebar();
+
+      const pathPrefix = routeType === 'colony' ? `/${colony.name}/` : '/';
+
+      const derivedPath = `${pathPrefix}${path}`;
```

Comment:
I'm guessing `/` is passed as the path for dashboard as it ends up with an extra / `colony.name//`

<img width="415" alt="Screenshot 2024-08-02 at 15 29 30" src="https://github.com/user-attachments/assets/577cbdb6-5db9-4976-9ebc-046618d6916d">

Reply:
Will fix!